api/src/main/java/org/openmrs/module/sync2/client/rest/RestClient.java

```diff
@@ -0,0 +1,38 @@
+package org.openmrs.module.sync2.client.rest;
+
+import org.openmrs.api.context.Context;
+import org.openmrs.module.sync2.client.rest.resource.Patient;
+import org.openmrs.module.sync2.client.rest.resource.RestResource;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+
+
+public class RestClient {
+
+    private static final String PATIENT_CATEGORY = "patient";
+
+    public Object getObject(String category, String url) {
+        final String username = Context.getAdministrationService().getGlobalProperty("sync2.user.login");
```

Comment:
It's not a good solution to fetch login and password from the client itself. because we want the client to be usable against different servers (eg. parent and child). Username and password should be passed to the client. 
Please check my latest commits, I'm passing username and pwd via construtor

Reply:
https://github.com/ubap/openmrs-module-sync2/commit/3e090fb8fbda551fcf0f9969df82b9de8ab7fd10

Reply:
Good point!
Each of child instance will have only one Masters OpenMRS instance. However, it's good to keep this data in the SyncClient.